src/biblebot/bot.py

```
@@ -612,14 +620,44 @@ def __init__(self, config, client=None):
+
+        if raw_trigger_mode is not None:
+            mode_str = str(raw_trigger_mode).lower().strip()
+            if mode_str in (
+                TRIGGER_MODE_DIRECT_ONLY,
+                TRIGGER_MODE_SMART,
+                TRIGGER_MODE_ANYWHERE,
+            ):
+                self.trigger_mode = TriggerMode(mode_str)
+            else:
+                logger.warning(
+                    f"Invalid trigger_mode '{raw_trigger_mode}', falling back to '{TRIGGER_MODE_DIRECT_ONLY}'"
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)